src/test/resources/scenario43/output.json

```diff
@@ -0,0 +1,156 @@
+{
+    "timestamp": "2015-05-01T00:00:00.000+02",
+    "matches": [
+      {
+        "system_id": 100,
+        "subscription_id": 2,
+        "product_id": 1299,
+        "cents": 50,
+        "confirmed": true
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 2,
+        "product_id": 1300,
+        "cents": 50,
+        "confirmed": true
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 1,
+        "product_id": 1625,
+        "cents": 50,
+        "confirmed": true
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 1,
+        "product_id": 1625,
+        "cents": 100,
+        "confirmed": false
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 1,
+        "product_id": 1878,
+        "cents": 50,
+        "confirmed": true
+      },
+      {
+        "system_id": 100,
+        "subscription_id": 1,
+        "product_id": 1878,
+        "cents": 100,
+        "confirmed": false
+      },
+      {
+        "system_id": 101,
+        "subscription_id": 2,
+        "product_id": 1299,
+        "cents": 0,
+        "confirmed": true
+      },
+      {
+        "system_id": 101,
+        "subscription_id": 2,
+        "product_id": 1300,
+        "cents": 0,
+        "confirmed": true
+      },
+      {
+        "system_id": 101,
+        "subscription_id": 1,
+        "product_id": 1625,
+        "cents": 0,
+        "confirmed": false
+      },
+      {
+        "system_id": 101,
+        "subscription_id": 1,
+        "product_id": 1625,
+        "cents": 0,
+        "confirmed": true
+      },
```

Comment:
Two times one with confirmed true the other with false? Why?

Reply:
I think these matches have different `groupId`. One match belongs to the scenario, where subscription `1` covers both `1625` and `1299`, the other match belongs to scenarion, where `1` covers only `1625`.

I'll verify this now. I also think it'd make sense to include `groupId` ids in the output.